invenio_records_resources/resources/record.py

```diff
@@ -53,6 +55,21 @@ class RecordResourceConfig(ResourceConfig):
                 description="The record has been deleted.",
             )
         ),
+        PIDDoesNotExistError: create_errormap_handler(
+            HTTPJSONException(
+                code=404,
+                description="The pid does not exist.",
+            )
+        ),
+        PIDUnregistered: create_errormap_handler(
+            HTTPJSONException(
+                code=404,
+                description="The pid is not registered.",
+            )
+        ),
+        PIDRedirectedError: create_pidredirected_error_handler(
+            route="/records/{pid_value}"
```

Comment:
normal: can this be replaced with `RecordResourceConfig.item_route` ?

Reply:
It is not needed so I removed it :)